lmdeploy/pytorch/check_env/dist.py

```diff
@@ -38,6 +39,13 @@ def check(self):
                               message='dp>1 requires distributed_executor_backend="ray". '
                               f'Get distributed_executor_backend={distributed_executor_backend}.')
 
+        if self.ep > 1:
+            if self.dp % self.ep != 0:
+                self.log_and_exit(mod_name='Dist',
+                                  message=f'ep>1 requires dp % ep == 0. Get dp={self.dp} and ep={self.ep}.')
+            if self.tp != 1:
```

Comment:
This is true for moe, not whole model, other module (attention, mlp etc) might have different config.